packages/fxa-payments-server/src/components/PlanDetails/index.scss

```diff
@@ -0,0 +1,116 @@
+.plan-details-component {
+  background: #fff;
+  border-radius: 8px;
+  box-shadow: 0 12px 18px 2px
+rgba(34, 0, 51, 0.04), 0 6px 22px 4px rgba(7, 48, 114, 0.12), 0 6px 10px -4px
+rgba(14, 13, 26, 0.12);
+  font-size: 15px;
+  margin: 0 auto;
+  padding: 20px 20px 0;
+  width: 342px;
+
+  .plan-details-header-wrap {
+    display: flex;
+  }
+
+  .plan-details-heading-wrap {
+    display: flex;
+    flex-direction: column;
+    padding-left: 10px;
+    text-align: left;
+
+    .plan-details-product {
+      /* Grey/Grey 50 */
+      color: rgba(12, 12, 13, 0.8);
+      font-weight: 600;
+      line-height: 19px;
+      word-wrap: break-word;
+    }
+
+    .plan-details-description {
+      color: rgba(12, 12, 13, 0.6);
+      margin-top: 4px;
+    }
+  }
+
+  header {
+    border-bottom: 1px solid#e7e7e7;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 0;
+    padding-bottom: 20px;
+  }
+
+  header p {
+    margin: 0;
+  }
+
+  header img {
+    display: block;
+    height: 64px;
+    width: 64px;
+  }
+
+  .plan-details-list {
+    border-bottom: 1px solid#e7e7e7;
+    color: rgba(12, 12, 13, 0.8);
+    margin-top: 10px;
+    text-align: left;
+
+    h4 {
+      margin: 16px 0;
+    }
+
+    ul {
+      color: #6d6d6e;
+      margin: 0;
+      padding-left: 15px;
+    }
+
+    li {
+      margin-bottom: 16px;
+
+      ::marker {
+        font-size: 12px;
+      }
+    }
+  }
+
+  .btn {
+    background: none !important;
```

Comment:
I know we do this in the auth-server email CSS, but we should _really_ avoid using `!important`. If we need to override this later we'll have to override selector specificity _and_ add `!important`. Can we achieve the same thing with selector specificity?

Reply:
We do it in the content-server as well and if it has to be there, a comment explaining why would be good.

Reply:
Yeah I just copied from content-server. From what I understand we only need to use `!important` here when we are resetting the default `<button>` styles. I can try to find a workaround, if I can't find one quick I'll just leave a comment